contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -221,8 +230,8 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
             );
         }
 
-        uint256 amount = BURNED_TX_FEES_ADDRESS.balance;
-        uint256 scaledAmount = _scaleTokens(amount, false);
+        uint256 burnedDifference = BURNED_TX_FEES_ADDRESS.balance - latestBurnedFeesReported;
```

Comment:
We need to increment `latestBurnedFeesReported` here. Lets also add a unit test that would catch this case by calling `reportBurnedTxFees` multiple times. 

I think we should require that the `scaledAmount` is non-zero also.